### ISSUE TITLE
[Link and button labels] Button and link language isn't descriptive and/or their purpose isn't clear. (08.03.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#67296]
#### VFS team name
MHV-Medications

#### VFS product name
MHV Medications

#### VFS feature name
Medications

#### Point of Contact/Reviewers
Ryan Smith - @rsmithadhoc - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Button and link language isn't descriptive and/or their purpose isn't clear.

### Issue Details
There are links on the page that do not contain any text. Screen readers have no text to read, plus this element can be tabbed to, which may cause confustion. This was found both by running the Axe browser extension and through keyboard testing by tabbing through interactive elements. Example: `<a class="vads-u-margin-y--0p5 vads-u-font-size--h4" data-testid="medications-history-details-link" href="/my-health/medications/21142597"></a>`

### Link, screenshot or steps to recreate
[Screenshot of missing link text](https://raw.githubusercontent.com/department-of-veterans-affairs/va.gov-team/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/58241_ryan_1.png)

### VA.gov Experience Standard
[Category Number 08, Issue Number 03](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.4.4_A

### Platform Recommendation
Ensure these links have text or remove the card if it should not be showing.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by TannerBird98** [comment]
This was fixed from an old ticket, all links have text on them in our current UI
